engine/audioEngine.js

```diff
@@ -0,0 +1,113 @@
+(function(LP) {
+
+  //Creates an audio element
+  function getSound(audioItem) {
+    var sound = document.createElement("audio");
```

Comment:
Wonder if creating an element for each sound doesn't create too much work on the garbage collector or something. Anyway, we could optimize it later if required ;)